src/main/java/fopbot/Direction.java

```diff
@@ -1,23 +1,155 @@
 package fopbot;
 
+import java.util.Arrays;
+
 /**
  * Defines the (viewing) direction that an entity can have.
  */
 public enum Direction {
+    /**
+     * The neutral direction or direction of view "none".
+     */
+    NONE(0, 0),
```

Comment:
What is the point of `NONE`? I don't see any advantage to adding this and the disadvantage of breaking modulo math for calculating directions

Reply:
agreed